modules/nw-egressnetworkpolicy-object.adoc

```diff
@@ -1,70 +1,156 @@
 // Module included in the following assemblies:
 //
 // * networking/openshift_sdn/configuring-egress-firewall.adoc
+// * networking/ovn_kubernetes_network_provider/configuring-egress-firewall.adoc
+
+ifeval::["{context}" == "openshift-sdn-egress-firewall"]
+:kind: EgressNetworkPolicy
+:api: network.openshift.io/v1
+:openshift-sdn:
+endif::[]
+ifeval::["{context}" == "ovn-configuring-egress-firewall"]
+:kind: EgressFirewall
+:api: k8s.ovn.org/v1
+:ovn:
+endif::[]
 
 [id="nw-egressnetworkpolicy-object_{context}"]
-= EgressNetworkPolicy custom resource (CR) object
+= {kind} custom resource (CR) object
 
-The following YAML describes an EgressNetworkPolicy CR object:
+You can define one or more rules for an egress firewall. A rule is either an `Allow` rule or a `Deny` rule, with a specification for the traffic that the rule applies to.
 
-[source,yaml]
+The following YAML describes an {kind} CR object:
+
+.{kind} object
+[source,yaml,subs="attributes+"]
 ----
-apiVersion: network.openshift.io/v1
-kind: EgressNetworkPolicy
+apiVersion: {api}
+kind: {kind}
 metadata:
   name: <name> <1>
 spec:
   egress: <2>
     ...
 ----
-<1> Specify a `name` for your egress firewall policy.
+<1> A name for your egress firewall policy.
```

Comment:
The EgressFirewall policy name only can be "default" for now, cannot use other name here,  refer to bug:https://bugzilla.redhat.com/show_bug.cgi?id=1862847